ast/Trees.cc

```diff
@@ -1188,4 +1188,18 @@ string BlockArg::nodeName() {
     return "BlockArg";
 }
 
+MaybeASTPassResult::MaybeASTPassResult() : trees(nullopt){};
+MaybeASTPassResult::MaybeASTPassResult(std::vector<ParsedFile> trees) : trees(move(trees)) {}
```

Comment:
Do you want to take a `vector<ParsedFile> &&trees` here? I don't really understand how `move` works in the context of `vector`.